config/initializers/assets.rb

```diff
@@ -8,9 +8,6 @@
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
 # Add Yarn node_modules folder to the asset load path.
+Rails.application.config.assets.precompile += %w(404.html 500.html)
```

Comment:
According to https://guides.rubyonrails.org/asset_pipeline.html#search-paths and https://guides.rubyonrails.org/asset_pipeline.html#precompiling-assets these lines shouldn't be necessary, but I'll believe it if you tell me otherwise.